### PR TITLE
Feat/api resources view

### LINKED ISSUE #343: Create a view for api-resources
much like alias menu on k9s
display the output of `kubectl api-resources` and allow clicking on a resource to see it
we can maybe use the cached_resources we already have

### PR DESCRIPTION
**Pull request by mosheavni**
View for api-resources fixes #343 
image

### CODE REVIEW

File: lua/kubectl/views/api-resources/init.lua [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+function M.View(cancellationToken)
+  local self = state.instance
+  if not self or not self.resource or self.resource ~= definition.resource then
+    self = ResourceBuilder:new(definition.resource)
+  end
+
+  self:display(definition.ft, definition.resource, cancellationToken)
+
+  state.instance = self
+  self.data = cached_resources and cached_resources.values or {}
```
Comment (Ramilito) [inline lua/kubectl/views/api-resources/init.lua:17]: cached_resources doesn't seem to be defined here ☝️

Comment (mosheavni) [reply lua/kubectl/views/api-resources/init.lua:18]: oops. fixed

File: ftplugin/k8s_api_resources.lua [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      view.view_or_fallback(name)
+    end,
+  })
+end
+
+--- Initialize the module
+local function init()
+  set_keymaps(0)
+  if not loop.is_running() then
+    loop.start_loop(api_resources_view.View)
```
Comment (Ramilito) [inline ftplugin/k8s_api_resources.lua:32]: Do we want to rerun view or draw in this loop?

Comment (mosheavni) [reply ftplugin/k8s_api_resources.lua]: yeah why not. changed to `Draw`

**Review by Ramilito — APPROVED** [review]

**Pull request merged**